server/fleet/software_installer.go

```diff
@@ -516,7 +522,8 @@ type SoftwarePackageOrApp struct {
 	PackageURL    *string                `json:"package_url"`
 	// InstallDuringSetup is a boolean that indicates if the package
 	// will be installed during the macos setup experience.
-	InstallDuringSetup *bool `json:"install_during_setup,omitempty" db:"install_during_setup"`
+	InstallDuringSetup *bool    `json:"install_during_setup,omitempty" db:"install_during_setup"`
+	Categories         []string `json:"categories,omitempty"`
```

Comment:
Why is this OmitEmpty? Guessing installer/VPP app having this not-omitempty means we'll always display the field name when needed?

Reply:
Right, iirc the field was showing up in the list software call where it wasn't specified